suites/manage/ocp/parallel-pv.py

```diff
@@ -0,0 +1,18 @@
+
+import os
```

Comment:
i think that suites folder was intended to just hold yaml suites files and not python code. I see that @ebenahar created the __init__ files here, but any helper functions for tests and tests should be under tests directory and we should create teams structure there I think.  @ebenahar I see that structure was done by you in #47 but I think that this structure should have been done under tests directory. For suites as I am not sure about usage of them when moved to pytest, but these shouldn't be probably python packages as I understand suites so far.